### PR TITLE
HPCC-20445 DFUPlus examples should use default settings

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@gfortil Please review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

http://10.240.32.243/view/Docs%207.0/job/JimDocBuild7/CE=ubuntu-14.04-amd64-1/lastSuccessfulBuild/artifact/build-docs/Release/docs/EN_US/HPCCClientTools_EN_US-7.0.0-rc2.pdf
Begins on page 70

### CODE REVIEW

File: docs/EN_US/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml [hunk 1]
```diff
@@ -553,26 +554,25 @@ dfuplus action=spray srcip=10.150.50.14
...
-// c:\import on 10.150.51.26 to single logical file
-          LE::imagedb:
-dfuplus action=spray srcip=10.150.51.26
-          srcfile=c:\import\*.jpg,c:\import\*.bmp
+// /var/lib/HPCCSystems/mydropzone/ on 10.150.51.26 to single logical file LE::imagedb
 
-dstcluster=le_thor dstname=LE::imagedb overwrite=1
+dfuplus action=spray srcip=10.150.51.26
+          srcfile=/var/lib/HPCCSystems/mydropzone/*.jpg,/var/lib/HPCCSystems/mydropzone/*.bmp
+          dstcluster=le_thor
```
Comment (gfortil) [inline docs/EN_US/HPCCClientTools/CT_Mods/CT_Comm_Line_DFU.xml]: could we use mythor as dstcluster by default?

**Review by gfortil — CHANGES_REQUESTED** [review]
My comments are inline.

**Comment by JamesDeFabia** [comment]
@gfortil Edits made, please have another look

**Review by gfortil — APPROVED** [review]
Looks good

**Comment by JamesDeFabia** [comment]
@richardkchapman I have squashed, this should be good to merge

**Pull request merged**